### DISCUSSION TITLE
20uF 1206 capacitors

### DISCUSSION THREAD

**Discussion by Tony359** [BlueSCSI/BlueSCSI-v2#154] [Build it Yourself]
Hello all,
I'm trying to buy the components to build my own BlueSCSI - yes, I know a kit is more convenient but I'd like to make my own. I couldn't find a pre-made Digikey/Mouser BOM online so I'm making my own.

A couple of questions
1. C3, C4 and C23 are 20uF caps, 1206 package. Digikey don't have 20uF for those. Closest value is 22uF which I assume is ok but I just wanted to check
2. are C23 and C3/C4 the same package? The footprint has a different description but they're both "1206". 
3. Same for C8 -- C24 and C17-C18: the footprint is 0805 for both but there are different measurements?

I'm sure I'll come up with more questions!
Thanks!

**Comment by androda** [comment]
1: 22uF is fine.  Any value between 22uF and 47uF should do just as well.
2: This is a frustration of mine, KiCAD keeps changing the naming of their component footprints between versions so components that are added later don't have the same footprint naming as earlier versions.
3: See above

**Reply by Tony359** [reply]
ah, I understand about the different naming - thanks!
Also thanks for the advice on the cap!
